sdk/instrumentation/instrumentation-opentelemetry/test/public/util/testTracer.ts

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I'm assuming we'd update the test-utils package after we finish abstracting OpenTelemetry out of the main repo? 

Would there be an equivalent set of "test" implementations for Instrumentation that we replace our existing TestSpan/TestTracer with?

Reply:
That's the current plan, yup! And once we are happy with test-utils we can go back and reuse those here (unless we end up with OTel specific tests we'll see...)